stix_shifter_modules/aws_athena/tests/stix_translation/test_aws_athena_json_to_stix.py

```diff
@@ -490,19 +491,53 @@ def test_ocsf_translation_prop(self):
         assert user_account['user_id'] == '011222333553'
         assert user_account['display_name'] == 'backup'
 
+        x_ibm_finding = TestAwsResultsToStix.get_first_of_type(objects.values(), 'x-ibm-finding')
+        assert x_ibm_finding is not None, 'x-ibm-finding object type not found'
+        assert x_ibm_finding.keys() == {'type', 'time_observed', 'dst_ip_ref', 'src_ip_ref', 'severity'}
+        assert x_ibm_finding['time_observed'] == '2020-10-07T08:08:37.000Z'
+        assert x_ibm_finding['severity'] == 0
+
+    def test_ocsf_file_translation(self):
```

Comment:
these 2 unittests (test_ocsf_file_translation and test_ocsf_network_translation) fail with error:
```
{'success': False, 'code': 'invalid_parameter', 'error': 'Error when converting STIX pattern to data source query: the JSON object must be str, bytes or bytearray, not list'}
```
My guess is the async aws athena connector processes the mapping file differently.